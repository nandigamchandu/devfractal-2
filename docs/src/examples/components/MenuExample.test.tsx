import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { MenuExample } from './MenuExample'

test('Menu Example', () => {
  const component: ReactTestRenderer = renderer.create(<MenuExample />)
  expect(component.toJSON()).toMatchSnapshot()
})
