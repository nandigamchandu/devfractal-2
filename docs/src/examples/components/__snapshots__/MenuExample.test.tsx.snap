// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Example 1`] = `
<section
  className="section"
>
  <div
    className="box"
  >
    <aside
      className="menu"
    >
      <p
        className="menu-label"
      >
        General
      </p>
      <ul
        className="menu-list"
      >
        <li
          className=""
        >
          <a>
            Dashboard
          </a>
        </li>
        <li
          className=""
        >
          <a>
            Customers
          </a>
        </li>
      </ul>
      <p
        className="menu-label"
      >
        Administration
      </p>
      <ul
        className="menu-list"
      >
        <li
          className=""
        >
          <a>
            TeamSettings
          </a>
        </li>
        <li
          className=""
        >
          <a
            className="is-active"
          >
             Manage your team
          </a>
          <ul
            className="menu-list"
          >
            <li
              className=""
            >
              <a>
                Members
              </a>
            </li>
            <li
              className=""
            >
              <a>
                Plugins
              </a>
            </li>
            <li
              className=""
            >
              <a>
                Add a member
              </a>
            </li>
          </ul>
        </li>
        <li
          className=""
        >
          <a>
            invitations
          </a>
        </li>
        <li
          className=""
        >
          <a>
            cloudstorage
          </a>
        </li>
        <li
          className=""
        >
          <a>
            invitations
          </a>
        </li>
        <p
          className="menu-label"
        >
          Transactions
        </p>
      </ul>
      <ul
        className="menu-list"
      >
        <li
          className=""
        >
          <a>
            payments
          </a>
        </li>
        <li
          className=""
        >
          <a>
            transfers
          </a>
        </li>
        <li
          className=""
        >
          <a>
            balance
          </a>
        </li>
      </ul>
    </aside>
  </div>
</section>
`;
